mlflow/environment_variables.py

```diff
@@ -186,3 +186,10 @@ def get(self):
 MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT = _EnvironmentVariable(
     "MLFLOW_SCORING_SERVER_REQUEST_TIMEOUT", int, 60
 )
+
+#: Specifies the backend store pagination size for the GetMetricHistory API
+#: Note: Pagination is only supported with `databricks` as a backend for this API
+#: (default: ``25000``)
+MLFLOW_GET_METRIC_HISTORY_MAX_RESULTS_PER_PAGE = _EnvironmentVariable(
+    "MLFLOW_GET_METRIC_HISTORY_MAX_RESULTS_PER_PAGE", int, 25000
+)
```

Comment:
Not sure this is necessary - when would users want to set it?

Reply:
Good point. It really should be a managed setting. I'll remove this :)